wwwapp/models.py

```diff
@@ -73,7 +115,7 @@ def __str__(self):
 
 class UserInfo(models.Model):
     """Info needed for camp, not for qualification."""
-    pesel = models.CharField(max_length=20, blank=True, default="")
+    pesel = PESELField(max_length=11, blank=True, default="")
```

Comment:
Changing max_length requires a migration

Reply:
Is that something I should include in the pull request? I honestly don't know enough Django.

Reply:
Yes - just run `manage.py makemigrations` and attach the generated file in the commit. This will generate a file that updates the field length in the database after you run `manage.py migrate`.